pytensor/tensor/extra_ops.py

```diff
@@ -1480,22 +1486,24 @@ def broadcast_shape_iter(
     arrays
         An iterable of tensors, or a tuple of shapes (as tuples),
         for which the broadcast shape is computed.
-    arrays_are_shapes
+    arrays_are_shapes: bool, default False
         Indicates whether or not the `arrays` contains shape tuples.
         If you use this approach, make sure that the broadcastable dimensions
         are (scalar) constants with the value ``1``--or simply the integer
-        ``1``.
+        ``1``. This is not revelant if `allow_runtime_broadcast` is True.
+    allow_runtime_broadcast: bool, default False
```

Comment:
I think we've (or at least I have) mostly been calling this dynamic broadcasting, but I actually like runtime broadcasting more. We just shouldn't mix them in the code base if this comes up in other places.

Reply:
Definitely! Let's stick with runtime broadcasting